docs/data-model-queries.md

```diff
@@ -2,15 +2,20 @@
 
 ## Indexes and Fields
 
-*Index* and *field*s are the main data models of Pilosa. You can check the [Pilosa documentation](https://www.pilosa.com/docs) for more detail about the data model.
+*Index* and *field*s are the main data models of Pilosa. You can check the [Pilosa documentation](https://www.pilosa.com/docs/latest/data-model/) for more detail about the data model.
 
-The `schema.Index` function is used to create an index object. Note that this does not create an index on the server; the index object simply defines the schema.
+The `schema.Index` function is used to create an index instance. Note that this does not create an index on the server; the index object simply defines the schema.
 
 ```go
 schema := pilosa.NewSchema()
 repository := schema.Index("repository")
 ```
 
+You can customize pass options while creating index instances:
```

Comment:
there's an extra word in here - maybe just remove "customize"

Reply:
updated